grpc/nodejs/BUILD

```diff
@@ -37,54 +38,26 @@ nodejs_binary(
   templated_args = ["--nobazel_node_patches"],
 )
 
+py_binary(
```

Comment:
what is this for @vmax ?

Reply:
how is this related to the declaration to `ts_grpc_compile` below?

Reply:
@lolski so the way we compile proto files is:

1) turn proto targets into their respective sources
2) use old genrule to generate js/ts files into one .tar.gz archive
3) use another rule which unpacks that archive into the files that we expect

(3) is covered by this `py_binary` target - it takes in an archive with generated sources and all of the filenames that are expected in the output and extracts them

Reply:
Is it not possible to extract the source files via the proto_library .src property? This came up in research, but I was unclear on the feasibility.

Reply:
@Phillammon rules and macros don't have access to target properties directly - only if targets choose to expose them via _providers_ (such as `ProtoInfo` in this case)